lnrpc/rpc.proto

```diff
@@ -802,6 +800,19 @@ message ChannelPoint {
     uint32 output_index = 3 [json_name = "output_index"];
 }
 
+message OutPoint {
+        oneof txid {
+                /// Raw bytes representing the transaction id.
+                bytes txid_bytes = 1 [json_name = "txid_bytes"];
+
+                /// Hex-encoded string representing the transaction id.
+                string txid_str = 2 [json_name = "txid_str"];
+        }
+
+        /// The index of the output on the transaction.
+        uint32 output_index = 3 [json_name = "output_index"];
+}
```

Comment:
Tabs off (8 instead of 4)?

Reply:
We have different tabwidths for protos? 🤦‍♂️ 

Starting to think we should commit a default vimrc in the tree...